pkgs/applications/networking/sync/lcsync/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  pkgs,
+  fetchFromGitea,
+  lcrq,
+  librecast,
+  lib,
+  ...
+}:
+stdenv.mkDerivation rec {
+  name = "lcsync";
+  version = "0.2.1";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "librecast";
+    repo = "lcsync";
+    rev = "v${version}";
+    sha256 = "sha256-RVfa0EmCPPT7ndy94YwD24S9pj7L11ztISaKHGcbTS8=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.